### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by jehawley**
Bump library version to avoid deprecated GitHub API.
BOT NOTES: 
resolves https://github.com/solo-io/build-bot/issues/73

### CODE REVIEW

File: botutils/botconfig/config.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func (r *configReader) updatePrivateKey(config *Config) error {
 func (r *configReader) updateIntegrationId(config *Config) error {
 	integrationIdStr := r.os.Getenv(IntegrationIdEnvVar)
 	if integrationIdStr != "" {
-		integrationId, err := strconv.Atoi(integrationIdStr)
+		integrationId, err := strconv.ParseInt(integrationIdStr, 10, 64)
```
Comment (kdorosh) [inline botutils/botconfig/config.go:144]: why this change?

File: go.mod [hunk 2]
```diff
@@ -25,11 +24,11 @@ require (
 	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
 	github.com/golang/mock v1.3.1
 	github.com/google/go-github v17.0.0+incompatible
```
Comment (kdorosh) [inline go.mod:26]: i would expect this to be removed?

File: botutils/botconfig/config.go (hunk 1 above)
Comment (jehawley) [reply botutils/botconfig/config.go:144]: If you look at the assignment below (`config.Github.App.IntegrationID = integrationId`), `IntegrationID` is now an int64.

File: go.mod (hunk 2 above)
Comment (jehawley) [reply go.mod]: Removed

File: githubutils/repo.go [hunk 3]
```diff
@@ -197,7 +197,7 @@ func DownloadRepoArchive(ctx context.Context, client *github.Client, w io.Writer
 		Ref: sha,
 	}
 
-	archiveURL, _, err := client.Repositories.GetArchiveLink(ctx, owner, repo, github.Tarball, opt)
+	archiveURL, _, err := client.Repositories.GetArchiveLink(ctx, owner, repo, github.Tarball, opt, true)
```
Comment (kdorosh) [inline githubutils/repo.go:200]: for those following along, this bool determines if we follow redirects. lgtm

**Review by kdorosh — APPROVED** [review]
